### PR TITLE
plumed: add recent versions

### PR DESCRIPTION
**Pull request by w8jcik**

### CODE REVIEW

File: var/spack/repos/builtin/packages/plumed/package.py [hunk 1]
```diff
@@ -21,17 +21,27 @@ class Plumed(AutotoolsPackage):
...
     homepage = 'https://www.plumed.org/'
-    url = 'https://github.com/plumed/plumed2/archive/v2.6.3.tar.gz'
+    url = 'https://github.com/plumed/plumed2/archive/v2.7.4.tar.gz'
     git = 'https://github.com/plumed/plumed2.git'
 
     version('master', branch='master')
+
+    version('2.8.0', sha256='8357eca6f280125037ad4e7c427f96f2af2f60ddfedce1a2e36e1e1cc3bff32b')
+
+    version('2.7.4', preferred=True, sha256='6f5dc16a753c71ed719f17f9cbb61642ab8e716fb7f77e22960dfa145c3b8211')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/plumed/package.py]: Why do we prefer v2.7.4?

**Review by alalazo — CHANGES_REQUESTED** [review]
Just a question, otherwise LGTM.

File: var/spack/repos/builtin/packages/plumed/package.py (hunk 1 above)
Comment (w8jcik) [reply var/spack/repos/builtin/packages/plumed/package.py]: I am just trying to mark latest stable version.

`2.6.6` is marked as _unsupported_ here https://www.plumed.org/download.

So I bumped preferred version to `2.7.x`.

I selected `2.7.4` over `2.8.0` for no special reason.

Comment (alalazo) [reply var/spack/repos/builtin/packages/plumed/package.py]: Ok, if there's no particular reason, can we remove the `preferred=True` and let Spack go with v2.8.0 as the default?

Comment (w8jcik) [reply var/spack/repos/builtin/packages/plumed/package.py]: Done

**Review by alalazo — APPROVED** [review]

**Pull request merged**